industrial_ci/src/docker.sh

```diff
@@ -135,6 +135,9 @@ function docker_cp {
 #######################################
 function ici_docker_build() {
   local opts=($DOCKER_BUILD_OPTS)
+  if [ "$DOCKER_PULL" != false ]; then
```

Comment:
By mistake users can pass `DOCKER_PULL` with invalid values (similar to #296). So it'd be nice for this `if` logic to be strict.

Reply:
For now I don't want  to check these cases, because we don't have the proper infrastructure (something like #285)

Reply:
Agreed. That is better than repeating to embed same/similar if-else logic.